.github/workflows/error-codes-updater.yaml

```diff
@@ -0,0 +1,65 @@
+name: Meshkit Error Codes Utility Runner
+on:
+  push:
+    branches:
+      - 'master'
+    paths:
+      - '**.go'
+
+jobs:
+  Update-error-codes:
+    name: Error codes utility
+    if: github.repository == 'layer5io/meshkit'
```

Comment:
```suggestion
    if: github.repository == 'meshery/meshery-consul'
```

Reply:
Sorry for that, I should have checked the names. 😅